From 7d29bde15c994cddd6feb52926991368e2d0ebf2 Mon Sep 17 00:00:00 2001
From: Danilo Spinella <danyspin97@protonmail.com>
Date: Tue, 16 Apr 2019 14:29:03 +0200
Subject: [PATCH] Respect cross compilation when calling pkg-config

---
 configure | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/configure b/configure
index 4b911a5..8762ff9 100755
--- a/configure
+++ b/configure
@@ -348,13 +348,19 @@ if [ "x$target" != "x$(cat $sysdeps/target)" ] ; then
   exit 1
 fi
 
+if ! test -z "$PKG_CONFIG" && command -v $PKG_CONFIG > /dev/null ; then
+    pkg_config=$PKG_CONFIG
+else
+    pkg_config=${cross}pkg-config
+fi
+
 spawn_lib=$(cat $sysdeps/spawn.lib)
 socket_lib=$(cat $sysdeps/socket.lib)
 sysclock_lib=$(cat $sysdeps/sysclock.lib)
 tainnow_lib=$(cat $sysdeps/tainnow.lib)
 timer_lib=$(cat $sysdeps/timer.lib)
 util_lib=$(cat $sysdeps/util.lib)
-procps_lib=$(pkg-config --libs libprocps)
+procps_lib=$($pkg_config --libs libprocps)
 
 if $allpic ; then
   tryflag CFLAGS_AUTO -fPIC
-- 
2.21.0

